### ISSUE TITLE
Problem getting the toolbox started 

### ISSUE THREAD

**Issue by fbarbey** [peclayson/ERA_Toolbox#26]
## Context
I just installed the toolbox and I tried to run some analyze for the first time and I got an error (pasted below). I just loaded up an excel file with my data with multiples columns including for my Subj ID and one with my numerical data to model. 
I also got a msg "Checking the installed version of MatlabStan is not currently supported" I am not sure of what it means 

## Screenshot
![image](https://user-images.githubusercontent.com/83653629/117014441-b55b9400-ace8-11eb-9fd4-020f27afebf6.png)

## Your Environment

* ERA Toolbox Version used: ERP Reliability Analysis Toolbox Version 0.5.2
* Matlab Version used: R2020b
* Operating System and Version (desktop or mobile): Windows 10
* CmdStan Version (if applicable): cmdstan-2.26.1
* MatlabStan Version (if applicable):  2.15.1.0
* MatlabProcessManager Version (if applicable): 0.5.1

Here is the error message:

This may take a while depending on the amount of data
Compile failed with exit value: 2
Error using StanModel/compile (line 906)
Informationÿ: impossible de trouver des fichiers pour le(s) modŠle(s) sp‚cifi‚(s).
C:/Users/Florentine/Desktop/ERA_toolbox/Temp_StanFiles/cmdstan04-May-2021.hpp:1:0: sorry, unimplemented: 64-bit mode not compiled in

make: *** [make/program:61: C:/Users/Florentine/Desktop/ERA_toolbox/Temp_StanFiles/cmdstan04-May-2021.exe] Error 1

Error in StanModel/sampling (line 736)
            self.compile();

Error in stan (line 138)
      fit = model.sampling(p.Unmatched);

Error in era_computerel (line 688)
        fit = stan('model_code', stan_in,...

Error in era_computerelwrap (line 178)
    REL = era_computerel('data',era_data.proc.data,...

Error in era_startproc>era_exec (line 1565)
era_data = era_computerelwrap('era_prefs',era_prefs,'era_data',era_data);
 
Error while evaluating DestroyedObject Callback.

**Comment by peclayson** [comment]
Hm, Windows 10 has been causing a lot of problems lately. I hope to try and figure out a work around this summer.

Look at the steps [here](https://github.com/peclayson/ERA_Toolbox/issues/24#issuecomment-802244274) and read up [here](https://github.com/peclayson/ERA_Toolbox/issues/25). I've tried to help people sort through this themselves, but I suspect people run out of steam. 

It's important than when you try the above steps, you specify where to install cmdstanr, so it is in a location that can be read by MATLAB and is not contained in the R libraries directory. e.g., `install_cmdstan(dir = 'C:\\Users\\Me\\Documents')`

Let me know how it goes. I'm happy to try to help. 

Sorry for the headache.

-Peter

**Comment by fbarbey** [comment]
Hi Peter, 

I eventually figured it out thanks to your links  ! I have not used R though but python to proceed with the installation. What I did first was to install RTools40 via python directly in my ERADependents folder using the following line:

python -m cmdstanpy.install_cxx_toolchain -d "C:\Users\Florentine\Documents\MATLAB\ERADependents"

then to install cmdstan, I did:

python -m cmdstanpy.install_cmdstan -d "C:\Users\Florentine\Documents\MATLAB\ERADependents" -- compiler

then things worked out. I do not know why the --compiler option is needed but someone mentioned it on another forum and it worked out....

After that I manually installed MatlabProcessManager in the folder and MatlabStan and I started running my analyses today. 

Best wishes, 

Florentine
